.github/workflows/binaries.yaml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
     - master
     - release
+    - embed-etcd
```

Comment:
@noonio This commit was not intended to be merged. I forgot to point that out to you.